src/aks-preview/azext_aks_preview/_natgateway.py

```diff
@@ -0,0 +1,44 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from .vendored_sdks.azure_mgmt_preview_aks.v2021_07_01.models import ManagedClusterNATGatewayProfile
+from .vendored_sdks.azure_mgmt_preview_aks.v2021_07_01.models import ManagedClusterManagedOutboundIPProfile
+
+
+def create_nat_gateway_profile(managed_outbound_ip_count, idle_timeout):
```

Comment:
I would appreciate it if you could add some unit test for the functions in this file.

Reply:
Added.